<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="Client">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>891</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="nickname_input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QPushButton" name="make_room">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>390</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>방 만들기</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>16</width>
       <height>371</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>370</y>
       <width>851</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="set_nickname">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>390</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>닉네임 설정</string>
     </property>
    </widget>
    <widget class="QListWidget" name="room_list">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>561</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QListWidget" name="accessor_list">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>256</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLabel" name="nickname">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="subject">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_start">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>게임시작</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>주제:</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="game_stack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QLineEdit" name="answer">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="question">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>22</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>질문 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>53</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>정답 :</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QPushButton" name="yes_bt">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>예</string>
       </property>
      </widget>
      <widget class="QPushButton" name="no_bt">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>아니요</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="set_subject">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>130</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QListWidget" name="question_list">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>251</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>20</width>
       <height>441</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="chat_list">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>261</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="chat">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>410</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
    <widget class="QListWidget" name="member_list">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>251</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QPushButton" name="invite">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>400</y>
       <width>71</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>초대목록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="member">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>400</y>
       <width>71</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>참여자보기</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>20</width>
       <height>441</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>400</y>
       <width>71</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
